portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Date;
+
+import com.google.sps.data.Comment;
+import com.google.sps.data.CommentBuilder;
+
+import com.google.gson.Gson;
+
+@WebServlet("/comments")
+public final class CommentsServlet extends HttpServlet {
+    private List <Comment> getCommentsList() {
```

Comment:
Same here I believe it should be parameterized by project.

Reply:
Fixed that on #14 ! Please have a look there!